paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewState.kt

```diff
@@ -34,7 +35,8 @@ internal sealed class CustomerSheetViewState(
     val shouldDisplayDismissConfirmationModal: Boolean
         get() = this is AddPaymentMethod &&
             paymentMethodCode == PaymentMethod.Type.USBankAccount.code &&
-            bankAccountResult is CollectBankAccountResultInternal.Completed
+            bankAccountResult is CollectBankAccountResultInternal.Completed &&
```

Comment:
Are we guaranteed to have the Financial Connections SDK as a dependency here or could this blow up at runtime?

Reply:
I tested this, and fortunately it does not blow up at runtime, but I have added the check just in case.